src/coreclr/jit/optcse.cpp

```diff
@@ -2275,6 +2356,10 @@ CSE_HeuristicRL::CSE_HeuristicRL(Compiler* pCompiler)
         m_greedy = true;
     }
 
+    // Stopping "parameter"
+    //
+    m_registerPressure = CNT_CALLEE_TRASH + CNT_CALLEE_SAVED;
```

Comment:
does it need to take floating points into account here?

Reply:
Historically we haven't, and there are very few FP cses (maybe 1% of all candidates).